openshift/template.yaml

```diff
@@ -414,126 +332,82 @@ objects:
             timeoutSeconds: 10
           volumeMounts:
             - mountPath: /results
-              name: pv-analyzes
+              name: analyzes
         restartPolicy: Always
         volumes:
-          - name: pv-analyzes
+          - name: analyzes
             persistentVolumeClaim:
-              claimName: claim1
+              claimName: analyzes
     test: false
     triggers:
     - type: ConfigChange
     - type: ImageChange
       imageChangeParams:
         automatic: true
         containerNames:
-          - thoth-result-api
-        from:
-          kind: ImageStreamTag
-          name: 'thoth-result-api:latest'
-
-- apiVersion: v1
-  kind: DeploymentConfig
-  metadata:
-    labels:
-      service: graphexp
-      app: graphexp
-    name: graphexp
```

Comment:
Why do we want to remove graphexp?

Reply:
GraphExp should be part of a website, not anything we deploy with the core service

Reply:
In that case, I would move it to a separate YAML file so if we would like to deploy it, we still have this s2i configuration.